node/test/toolrunnertests.ts

```diff
@@ -395,8 +395,68 @@ describe('Toolrunner Tests', function () {
                 done(err);
             });
     })
+    // function to compile a .NET program that prints lines.
```

Comment:
we do a similar thing already to test the tool runner passing args correctly, so i reused that pattern here. this compiles a .net app on the fly using csc.